package.json

```diff
@@ -40,6 +40,6 @@
     "nodemon": "^1.19.4",
     "path": "^0.12.7",
     "session-file-store": "^1.4.0",
-    "xero-node": "^4.15.0"
+    "xero-node": "/Users/rett.behrens/Desktop/code/sdks/xero-node/xero-node-4.17.0.tgz"
```

Comment:
Remove local reference for master release

Reply:
> I don't see any example of the request financial statements by contact features. Was that a conscious decision?

Nope. Good catch. This branch dates back to before they were added I think. Will add them now. Thanks!